app/code/Magento/EncryptionKey/README.md

```diff
@@ -1 +1,9 @@
+#Magento_EncryptionKey module
```

Comment:
Please include the Extensibility section (https://github.com/magento/devdocs/wiki/Magento-module-README.md#extensibility) with the Layout subsection containing an information that this module introduces a new layout handle - `adminhtml_crypt_key_index`